src/ClientInterface.php

```diff
@@ -10,7 +10,7 @@
     /**
      * Sends a PSR-7 request and returns a PSR-7 response.
      *
-     * Every technically correct HTTP response MUST be returned as is, even if it represents an HTTP
+     * Every technically correct HTTP response MUST be returned as is, even if it represents a HTTP
```

Comment:
You sure?

https://www.quora.com/A-http-or-an-http-which-is-the-correct-grammatical-usage

Reply:
Im sure I **copied** it right, but I think I agree with you on the grammar. 

https://github.com/php-fig/fig-standards/blob/master/proposed/http-client/http-client.md#clientinterface

I do not think it is a blocker for the package. Could you please correct this at the specification?

Reply:
Done #1006